pupil-spa/src/environments/environment.ts

```diff
@@ -19,5 +19,6 @@ export const environment = {
   // Minimum display time for submission pending view
   submissionPendingViewMinDisplay: 2000,
   // Contact number in case check submission fails
-  supportNumber: '0345 278 8080'
+  supportNumber: '0345 278 8080',
+  googleAnalyticsTrackingCode: 'UA-92976848-11'
```

Comment:
Is this a non-prod env?

Reply:
is non-prod and needs to be changes
@GuyHarwood any thoughts on the dev value of googleAnalyticsTrackingCode?

Reply:
As per our discussion- it needs to be disabled by default, and only active when a tracking code is present in the environment file. When deployed it will be built in production mode, so needs to be configured as such

Reply:
Empty string works for me.

Reply:
@GuyHarwood done